package/windows/Dockerfile.agent

```diff
@@ -2,7 +2,7 @@ ARG SERVERCORE_VERSION
 ARG ARCH=amd64
 ARG VERSION=dev
 
-FROM library/golang:1.23 as build
+FROM library/golang:1.23.5 as build
```

Comment:
Make sure we build the windows agent with the same toolchain version.

Reply:
Keeping the minor here, would mean we always have the latest patch release. Any specific reason we need to align patch versions?

Reply:
Ideally we should build all components with exactly the same Go version. If we don't set the patch version here, than from the time the new patch version comes out and until the bci image is updated (which ranges from days to weeks) the windows system agent will be built with a different patch version of Go.